pkg/gardenadm/cmd/init/init.go

```diff
@@ -259,12 +259,24 @@ To start using your cluster, you need to run the following as a regular user:
   sudo cp -i %s $HOME/.kube/config
   sudo chown $(id -u):$(id -g) $HOME/.kube/config
   kubectl get nodes
+`, botanist.PathKubeconfig)
+
+	for _, worker := range b.Shoot.GetInfo().Spec.Provider.Workers {
+		if worker.ControlPlane == nil {
+			fmt.Fprintf(opts.Out, `
+You can now join any number of worker machines to pool %q (or any other
+worker pool). Run this on a control plane node:
 
-You can now join any number of machines by running the following on each node
-as root:
+  gardenadm token generate --print-join-command --join-command-worker-pool-name %s
 
-  gardenadm join <TODO>
+Copy the output and run it as root on each node you would like to join the
+cluster.
+`, worker.Name, worker.Name)
```

Comment:
Instead of passing `worker.Name` twice, you could use `%[1]s` (https://pkg.go.dev/fmt#hdr-Explicit_argument_indexes).